components/compliance-service/ingest/ingestic/project_update.go

```diff
@@ -0,0 +1,128 @@
+package ingestic
+
+import (
+	"context"
+	"strings"
+
+	iam_v2 "github.com/chef/automate/api/interservice/authz/v2"
+	"github.com/chef/automate/components/compliance-service/ingest/ingestic/mappings"
+	project_update_lib "github.com/chef/automate/lib/authz"
+	"github.com/chef/automate/lib/elasticutil"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func (backend *ESClient) ListProjectUpdateTasks(ctx context.Context) ([]project_update_lib.SerializedProjectUpdateTask, error) {
+	indexNames, err := backend.client.IndexNames()
+	if err != nil {
+		return nil, err
+	}
+
+	tasks := make([]project_update_lib.SerializedProjectUpdateTask, 0, 120)
+	for _, indexName := range indexNames {
+		var mapping mappings.Mapping
+
+		if strings.HasPrefix(indexName, mappings.ComplianceSumDate.Index) {
+			mapping = mappings.ComplianceSumDate
+		} else if strings.HasPrefix(indexName, mappings.ComplianceRepDate.Index) {
+			mapping = mappings.ComplianceRepDate
+		} else {
+			continue
+		}
+
+		indexDay, err := mapping.IndexTimeseriesScanDate(indexName)
+		if err != nil {
+			logrus.WithField("op", "ListProjectUpdateTasks").Warnf("Could not parse date from %s", indexName)
+			continue
+		}
+		priority := indexDay.Unix()
+		params := map[string]string{
+			"n": indexName,
+		}
+
+		tasks = append(tasks, project_update_lib.SerializedProjectUpdateTask{
+			Priority: priority,
+			Params:   params,
+		})
+	}
+
+	return tasks, nil
+}
+
+func (backend *ESClient) RunProjectUpdateTask(ctx context.Context, projectUpdateID string,
```

Comment:
was this code moved from somewhere else? what was running the project update before?

Reply:
This is new code. Search for `UpdateProjectTags` for the old code